social-network/src/ArticleCard.js

```diff
@@ -0,0 +1,123 @@
+                </div>
+            );
+        }
+        else {
+            return(
+                <div>
+                    {props.article.memberPreview && <FontAwesomeIcon icon={fasStar} style={{display: "block", margin: 2, color: "gray", float: "left"}}/>}
+                    {props.article.memberPreview && <Card.Text style={{color : "gray"}}>Member Preview</Card.Text>}
+                </div>
+            )
+        }
+    }
+
+    render() {
+        const date = new Date(this.props.article.postedDate);
+        const forYouStyle= {
+            width: 39 + 'rem',
+            height: 20 + 'rem',
+            marginTop:'2rem',
+            marginLeft:'2rem',
+            display: 'flex',
+            flexDirection: "row"
+        };
+
+        const forYouImgStyle= {
+            width : 15 + 'rem', 
+            objectFit: 'cover'
+        }
+
+        const forYouBodyStyle= {
+            width: 15 + 'rem',
+            display: 'flex',
+            flexDirection: 'column'
+        }
+
+        const forYouContentStyle= {
+            width: 15 + 'rem',
+            height: 15 + 'rem',
+            display: 'flex',
+            flexDirection: 'column'
+        }
+
+        const missingStyle= {
+            width: 25 + 'rem',
+            height: 22 + 'rem',
+            marginTop:'2rem',
+            marginLeft:'2rem',
+            display: 'flex',
+            flexDirection: "column"
+        }
+
+        const missingImgStyle= {
+            width: 25 + 'rem',
+            height: 7 + 'rem',
+            objectFit: 'cover'
+        }
+
+        const missingBodyStyle= {
+            width: 25 + 'rem',
+            height: 10 + 'rem',
+            display: 'flex',
+            flexDirection: 'column'
+        }
+
+        const missingContentStyle= {
+            width: 25 + 'rem', 
+            height: 18 + 'rem',           
+            display: 'flex',
+            flexDirection: 'column'
+        }
+        
+        return (            
+            <div id="ArticleCard">
+                
+                <Card style={this.props.imgTop ? missingStyle : forYouStyle}>
+                    <Card.Img style={this.props.imgTop ? missingImgStyle : forYouImgStyle} src={this.props.article.image} />
+                    <Card.Body style={this.props.imgTop ? missingBodyStyle : forYouBodyStyle}>
+                    {this.showMetaData(this.props)}
+                    <div style={this.props.imgTop ? missingContentStyle : forYouContentStyle}>
+                        <Card.Title>{this.props.article.title}</Card.Title>                       
+                        <Card.Text>
+                            {this.props.article.description}
+                        </Card.Text>
+                    </div>
+                    <div>
+                        <Image src={this.props.article.author.image} fluid thumbnail roundedCircle style={{float: "left", position: "relative"}}/>                        
```

Comment:
Try to avoid too many inline styles, defining your styles above the render is a good idea.